Detections/SecurityEvent/NonDCActiveDirectoryReplication.yaml

```diff
@@ -0,0 +1,56 @@
+id: b9d2eebc-5dcb-4888-8165-900db44443ab
+name: Non Domain Controller Active Directory Replication
+description: |
+  'This query uses Windows Event ID 4662 in order to detect potential attempts to synz an active directory object leveraging directory replication services (DRS)
+  from a non-domain-controller account with privileged permissions to do so. Ref: https://threathunterplaybook.com/notebooks/windows/06_credential_access/WIN-180815210510.html'
+severity: High
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvents
+queryFrequency: 1d
+queryPeriod: 7d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Credential Access
+relevantTechniques:
+  - T1003
+query: |
+  let timeframe = 7d;
+  // Enter a reference list of hostnames for your DC servers
+  let DCServersList = dynamic (["DC01.simulandlabs.com"]);
```

Comment:
Just to be safe, maybe include another example domain so they know the syntax of creating the list.  
```dynamic (["DC01.simulandlabs.com", "DC02.simulandlabs.com"])```

Reply:
fixed